packages/squiggle-lang/__tests__/TS/Jstat_test.ts

```diff
@@ -0,0 +1,45 @@
+import { errorValueToString } from "../../src/js/index";
+import * as fc from "fast-check";
+import { testRun } from "./TestHelpers";
+
+describe("Jstat: cumulative density function", () => {
```

Comment:
We're smoke testing our codepaths/dispatch but numerically we're validating that `Jstat` is doing what we expect. I put it in the describe so that when it fails you know what file to check (but come to think of it `jest` failures also point you to what file it's in, so you're right. I'll probably remove `Jstat` from the `describe` string on the next push.